Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py

```diff
@@ -483,6 +505,34 @@ def submit_an_inquiry_to_request_a_report_on_misjudgement_by_abnormal_security_c
     return command_results
 
 
+def submit_false_negative_report_command(client, args):
+    recipient_email = str(args.get('recipient_email;', ''))
+    sender_email = str(args.get('sender_email', ''))
+    subject = str(args.get('subject', ''))
+    response = client.submit_false_negative_report_request(recipient_email, sender_email, subject)
+    command_results = CommandResults(
+        outputs_prefix='AbnormalSecurity.SubmitFalseNegativeReport',
+        outputs_key_field='',
```

Comment:
```suggestion
```

Reply:
modified in pr: https://github.com/demisto/content/pull/20181/commits/2a60ec5d18ed2c81a381c97420b1d39339106f51